torch/fx/passes/runtime_assert.py

```diff
@@ -172,7 +172,10 @@ def _node_metadata_hook(
             node.args,
         )
         try:
-            node.meta[val_key] = node.target(*fake_args)  # type: ignore[operator]
+            target = node.target
+            if isinstance(node.target, str):
+                target = getattr(torch.Tensor, node.target)
```

Comment:
Nah, don't do this. The problem is sometimes the node is a call_method? The logic here should look something like doing an interpreter step on the FX node, which means we ought to be branching on node.op. But what is *definitely* wrong is hardcoding torch.Tensor as the getattr base here. cc @bobrenjc93 for other thoughts

Reply:
In practice, I see a node where `node.op == "call_method"` and `node.target == "storage_offset"`. It seems like the use of a string target breaks an assumption here.